### PR TITLE
Added operate and quotes methods

### PR DESCRIPTION
**Pull request by samuelkinyanjui**
Used the following external API: [API Ninjas](https://api-ninjas.com/api/quotes)
Created a new component for displaying a quote
Fetch data inside of a new component
Show a loading state
Show an error state
Import the new component into the main component

### CODE REVIEW

File: src/Quotes/Quote.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+        const json = await res.json();
+        setData(json[0]);
+      } catch (error) {
+        setHasError(true);
+      }
+      setIsLoading(false);
+    };
+    fetchData();
+  }, [setData, setIsLoading]); // eslint-disable-line react-hooks/exhaustive-deps
+  if (hasError) return <div className="quotes">Something went wrong!</div>;
```
Comment (Strangeal) [inline src/Quotes/Quote.js]: - [ ] Great job so far fetching data from the API  and making use of `useState` to its state so beautifully. However, Kindly consider fixing any `eslint` errors that may show up in your codebase and avoid disabling `eslint` as it is designed to help you write clean and industry-standard code. Instead

File: src/Quotes/Quote.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+      </span>
+    </div>
+  );
+}
+
+export function Footer() {
+  return (
+    <></>
+  );
+}
```
Comment (Strangeal) [inline src/Quotes/Quote.js]: - [ ] Kindly consider removing the `Footer` function from your codebase as you're not using it and it's a bad practice to leave unused code in your codebase as it make your code unclean and difficult to maintain.

**Review by Strangeal — APPROVED** [review]
Hi @samuelkinyanjui, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- Good **PR** title ✅ 
- Descriptive *PR* description ✅ 
- Implement the features ✅ 
- Nice *UI* design ✅ 
- Pixel Perfect Design ✅ 
- Professional `README` template ✅ 
- Correct `GitHub` workflow ✅ 
- Meaningful **Github** commits ✅ 
- Correct `.github/workflows` for linters ✅ 
- Followed *best-practices* ✅ 
- Correct **Semantics** used ✅ 
- Well structured project ✅ 

## Required Changes ♻️
- [ ] Impressive work so far. However, upon opening your project I noticed the data you're fetching from the `API` wasn't displaying on the UI so I quickly went through your code carefully and noticed you forgot to import the `Quote` component into the main component(`App.js`). To help users see the data you're fetching from the `API` kindly consider importing your `Quote` component inside your `App.js`

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/Quotes/Quote.js [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+        const json = await res.json();
+        setData(json[0]);
+      } catch (error) {
+        setHasError(true);
+      }
+      setIsLoading(false);
+    };
+    fetchData();
+  }, [setData, setIsLoading]); // eslint-disable-line react-hooks/exhaustive-deps
```
Comment (Efiamotu-1) [inline src/Quotes/Quote.js]: - [ ] In this [review link](https://github.com/samuelkinyanjui/mathmagicianone/pull/4#discussion_r1306383105), the previous reviewer requested that you should not disable eslint, however you re yet to apply this change kindly do so 👍

File: src/Quotes/Quote.js (hunk 2 above)
Comment (Efiamotu-1) [inline src/Quotes/Quote.js]: - In this [review link](https://github.com/samuelkinyanjui/mathmagicianone/pull/4#discussion_r1306383776), the previous reviewer asked you to remove unused code but you still have these codes displaying in your code base, can you kindly remove them as it is unprofessional 👍

**Review by Efiamotu-1 — CHANGES_REQUESTED** [review]
Hi @samuelkinyanjui , 

 

Good job so far! 👏. Your Code is Awesome! 💪

There are some **few** issues 🐛 that you still need to work 🔧 on though to go to the next project ⏭️ but you are almost there! 🏃
## Don't Worry 🙏

It's ok to have bugs in your code 🐛. Please, see this code review as a **chance** for you to **learn** practical coding skills and learn **new tips** that can improve your code quality 💯 and help you **get a better ** 💼.

_I am going to guide you on how to fix them down below. 🤓_

## Highlights :dart: 
- ✔️ No linter Errors
- ✔️ Descriptive README
- ✔️ Descriptive PR title and description
- The code is top-notch! 💯 and has a huge potential.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


## Important Notes 📝

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

> **Note:** Please, don't Merge This Pull Request yet. Only Merge when it is Approved ✅.


## Thank you for your time 🙏 and call me if you need help or clarification ☎️

- This is My **GitHub**: [@Efiamotu-1](https://github.com/Efiamotu-1/)
- This is My **Twitter**: [@EFYAMOTU](https://twitter.com/EFYAMOTU)
- This is My **linkedIn**.[Habeeb Efiamotu Musa](https://www.linkedin.com/in/musa-habeeb/)
- This is My **Slack**: [Habeeb Efiamotu Musa](https://slack.com)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/Quotes/Quote.js (hunk 3 above)
Comment (samuelkinyanjui) [reply src/Quotes/Quote.js]: Dear Efiamotu this was optional, not mandatory

File: src/Quotes/Quote.js (hunk 2 above)
Comment (samuelkinyanjui) [reply src/Quotes/Quote.js]: Dear Efiamotu i have used the footer to display on my page rather than deleting it. Also note it was optional comment

**Review by elisha2kyakpo1 — APPROVED** [review]
Hi 👋 @samuelkinyanjui, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

👍 You did a great job fixing the eslint error and removing the unused code as requested by previous reviewers.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**